test/coverage.yaml

```diff
@@ -15,7 +15,7 @@ directories:
   source/common/json: 95.2
   source/common/matcher: 94.7
   source/common/memory: 74.5  # tcmalloc code path is not enabled in coverage build, only gperf tcmalloc, see PR#32589
-  source/common/network: 94.4  # Flaky, `activateFileEvents`, `startSecureTransport` and `ioctl`, listener_socket do not always report LCOV
+  source/common/network: 94.3  # Flaky, `activateFileEvents`, `startSecureTransport` and `ioctl`, listener_socket do not always report LCOV
```

Comment:
afaict this is a real deal cov flake - i had a theory, and it worked initially, and then i realized it was more like i was caning the cache and rolling the dice